prow/config.yaml

```diff
@@ -219,7 +219,22 @@ tide:
   merge_label: tide/merge-method-merge
   context_options:
     # Treat unknown contexts as optional
-    skip-unknown-contexts: true
+    skip-unknown-contexts: false
+#     orgs:
```

Comment:
I assume that this would be `context_options.orgs` when un-commented?

Reply:
From the docs `If we want to skip unknown checks (ie checks that are not defined in Prow Config), we can set skip-unknown-contexts to true. This option can be set globally or per org, repo and branch.` In this case it is global.